test/language/arrow-function/lexical-bindings-overriden-by-formal-parameters-non-strict.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.2.16
+description: >
+    Runtime Semantics: Evaluation
+
+flags: [noStrict]
```

Comment:
Sorry, I completely misread what this was testing. I'll pay more attention from now on.